pubovl.py

```diff
@@ -18,11 +18,10 @@
 codeauthors: VierEck., DryByte (https://github.com/DryByte)
 '''
 
+from piqueserver.commands import command, target_player
+from pyspades.common import Vertex3, make_color
+from pyspades.constants import WEAPON_TOOL
```

Comment:
after changing some stuff to get this new version working, i reverted the change on line 67/68 thus i still needed to import WEAPON_KILL